diff --git a/WebKit/android/plugins/ANPKeyCodes.h b/WebKit/android/plugins/ANPKeyCodes.h
index edfe2b9..6246865 100644
--- a/WebKit/android/plugins/ANPKeyCodes.h
+++ b/WebKit/android/plugins/ANPKeyCodes.h
@@ -144,6 +144,8 @@ enum ANPKeyCodes {
     kButtonStart_ANPKeyCode = 108,
     kButtonSelect_ANPKeyCode = 109,
     kButtonMode_ANPKeyCode = 110,
+    kButtonStylus_ANPKeyCode = 111,
+    kButtonStylus2_ANPKeyCode = 112,
 
     // NOTE: If you add a new keycode here you must also add it to several other files.
     //       Refer to frameworks/base/core/java/android/view/KeyEvent.java for the full list.
